android/app/src/main/java/com/example/viewport/ui/ViewportScreens.kt

```
@@ -0,0 +1,943 @@
+        FlowRow(horizontalArrangement = Arrangement.spacedBy(8.dp), verticalArrangement = Arrangement.spacedBy(8.dp)) {
+          MetaChip("${project.galleryCount} galleries")
+          MetaChip("${project.totalPhotoCount} photos")
+          MetaChip(formatBytes(project.totalSizeBytes))
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun ProjectScreen(state: AppUiState, viewModel: ViewportViewModel, projectId: String) {
```

detekt (1.23.8)

[warning] The function ProjectScreen appears to be too complex based on Cognitive Complexity (complexity: 37). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)